docs/eventing/sources/sources.yaml

```diff
@@ -119,11 +119,17 @@ sources:
     description: >
       Brings [Apache Kafka](https://kafka.apache.org/) messages into Knative.
   - name: Apache CouchDB
-    url: https://github.com/knative/eventing-contrib/tree/{{< branch >}}/couchdb
+    url: https://github.com/knative/eventing-contrib/blob/master/couchdb/source/pkg/apis/sources/v1alpha1/couchdbsource_types.go
```

Comment:
I wonder if for consistency we should be pointing to the "top level" readme for couchdb instead of the actual types?
https://github.com/knative/eventing-contrib/tree/master/couchdb

Reply:
Other url like VMware what this format, but I am good with the README :) Thanks